include/wlr/render.h

```diff
@@ -32,13 +32,15 @@ struct wlr_texture *wlr_render_texture_create(struct wlr_renderer *r);
  * 	struct wlr_texture *texture;
  * 	float projection[16];
  * 	float matrix[16];
+ * 	float alpha;
```

Comment:
`alpha` isn't used.

Reply:
What do you mean by not used? It's in the wlr_render_with_matrix.

Reply:
This small example defines a few things used in the calls bellow.

I can see a literal `0.5f` at a place a variable named alpha would make sense though.
All other literals are inline, so I don't think it would make sense to declare and set alpha here.